src/index.js

```diff
@@ -5,9 +5,7 @@ import App from './App';
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
-  <React.StrictMode>
```

Comment:
I think you can revert this changes now, `StrictMode` proposes more intelligible solutions